docs/intro_07_bug_bounty.md

```diff
@@ -22,21 +22,27 @@ Many of the [Ethereum Foundation’s bug bounty program rules](https://bounty.et
 
 ## Scope
 
-The scope of our bug bounty program includes core contracts related to release *v1.1.1* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.1.1), [readme](https://github.com/gnosis/safe-contracts/blob/v1.1.1/README.md)) and *v1.2.0* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.2.0), [readme](https://github.com/gnosis/safe-contracts/blob/v1.2.0/README.md)) of the Gnosis Safe contracts.
+The scope of the bug bounty program includes the core contracts related to the following releases of the Gnosis Safe contracts:
+
+- *v1.1.1* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.1.1), [readme](https://github.com/gnosis/safe-contracts/blob/v1.1.1/README.md)) 
+- *v1.2.0* ([Release details](https://github.com/gnosis/safe-contracts/releases/tag/v1.2.0), [readme](https://github.com/gnosis/safe-contracts/blob/v1.2.0/README.md)) 
+
+The scope of the bug bounty also includes the [allowance module](https://github.com/gnosis/safe-modules/tree/master/allowances).
```

Comment:
Maybe we should mention the version here and link to the commit with that version directly ... for this stuff I am not really for using the `master` link as this theoretically might change. Maybe we should create a release for this on github